src/CodeGen_LLVM.cpp

```diff
@@ -2956,7 +2944,26 @@ void CodeGen_LLVM::visit(const Call *op) {
                 }
             }
         }
-
+    } else if (op->is_intrinsic(Call::saturating_add) || op->is_intrinsic(Call::saturating_sub)) {
+        internal_assert(op->args.size() == 2);
+        std::string intrin;
+        if (op->type.is_int()) {
+            intrin = "llvm.s";
+        } else {
+            internal_assert(op->type.is_uint());
+            intrin = "llvm.u";
+        }
+        if (op->is_intrinsic(Call::saturating_add)) {
+            intrin += "add.sat.";
+        } else {
+            internal_assert(op->is_intrinsic(Call::saturating_sub));
+            intrin += "sub.sat.";
+        }
+        if (op->type.lanes() > 1) {
```

Comment:
I don't understand what happens with non-native-width vectors here.

Reply:
Let's consolidate on the comment below, I think it's the same point.